<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>itemreport</class>
 <widget class="QWidget" name="itemreport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Cc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Cc:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Qty">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Qty:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Amt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Amt:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Ret">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Ret:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Disc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Disc</string>
   </property>
  </widget>
  <widget class="QLabel" name="Cost">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Cost</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CcValue">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="QtyValue">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="AmtValue">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="RetValue">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="DiscValue">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="CostValue">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ReportName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ReportName</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
